### PR TITLE
[Docs] Remove migrating to webpack page from docs

### PR DESCRIPTION
**Pull request by jakubtobiasz**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.12                   |
| Bug fix?        | no                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no  |
| Related tickets |                       |
| License         | MIT                                                          |

Initially, I was planning to make a bigger section about our changes in frontend but I think the section from the `UPGRADE` file is more than enough (ref https://github.com/Sylius/Sylius/pull/14330).

### CODE REVIEW

**Review by CoderMaggie — APPROVED** [review]

File: docs/cookbook/frontend/map.rst.inc [hunk 1]
```diff
@@ -1,3 +1,2 @@
 * :doc:`/cookbook/frontend/admin-js-and-css`
 * :doc:`/cookbook/frontend/google-analytics`
-* :doc:`/cookbook/frontend/webpack`
```
Comment (GSadee) [inline docs/cookbook/frontend/map.rst.inc:3]: You need also to remove this file from `docs/cookbook/index.rst`

**Review by GSadee — COMMENTED** [review]
Check if there is any other documentation that needs to be adapted, I mean mainly template files and assets paths

File: docs/cookbook/frontend/map.rst.inc (hunk 1 above)
Comment (jakubtobiasz) [reply docs/cookbook/frontend/map.rst.inc:3]: I've missed this one in `index.rst` as it was `frontend/webpack` (without `/cookbook`) :D.

**Comment by jakubtobiasz** [comment]
I've rechecked and there are no mentions about node version or any related stuff so this PR is complete for me.

**Review by GSadee — COMMENTED** [review]
As this is a PR for adjusting our docs after making webpack the default build tool, IMO you should adjust also [getting started](https://docs.sylius.com/en/latest/getting-started-with-sylius/installation.html#project-setup), [installation guide](https://docs.sylius.com/en/latest/book/installation/installation.html#installing-assets) and some places where assets are used as for example: https://docs.sylius.com/en/latest/getting-started-with-sylius/shop-customizations.html#logo

File: docs/book/frontend/managing-assets.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+If you want to add the ``logo.png`` to your build process, you can do it by importing your ``logo.png`` in your ``entry.js`` file:
+
+.. code-block:: javascript
+
+    // assets/shop/entry.js
+    import logo from './images/logo.png';
+
+    // ...
+
+After building, your image (on a default settings) should be available under ``<project_root>/public/build/app/shop/images/logo.<hash>.png`` path.
```
Comment (GSadee) [inline docs/book/frontend/managing-assets.rst]:
```suggestion
After building, your image (with default settings) should be available under ``<project_root>/public/build/app/shop/images/logo.<hash>.png`` path.
```

File: docs/book/frontend/managing-assets.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+Using assets in Twig templates
+------------------------------
+
+So far, we have seen how to import assets in JavaScript files. But what if you want to use them in Twig templates? The answer is simple: use the ``asset()`` function.
+
+.. code-block:: twig
+
+    <img src="{{ asset('build/app/shop/images/logo.png', 'app.shop') }}" />
+
+So far, when using ``asset()`` function you have been passing only one argument. Using Webpack we need to pass second arguments pointing to our package name from ``framework.assets.package`` configuration.
```
Comment (GSadee) [inline docs/book/frontend/managing-assets.rst]:
```suggestion
So far, when using ``asset()`` function you have been passing only one argument. Using Webpack we need to pass a second argument pointing to our package name from ``framework.assets.package`` configuration.
```

File: docs/book/frontend/overview.rst [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    │   ├── admin
+    │   │   ├── entry.js
+    │   ├── shop
+    │   │   ├── styles
+    │   │   │   ├── app.scss
+    │   │   ├── images
+    │   │   │   ├── logo.png
+    │   │   ├── entry.js
+
+If you want to know how import and managing those assets take a look at our :doc:`/book/frontend/managing-assets` guide.
```
Comment (GSadee) [inline docs/book/frontend/overview.rst]:
```suggestion
If you want to know how to import and manage those assets take a look at our :doc:`/book/frontend/managing-assets` guide.
```

File: docs/book/index.rst [hunk 5]
```diff
@@ -116,10 +116,22 @@ This part of the documentation is about the currently developed unified API for
 
 .. include:: /book/api/map.rst.inc
 
+Frontend
+------
```
Comment (GSadee) [inline docs/book/index.rst]:
```suggestion
--------
```

File: docs/book/index.rst [hunk 6]
```diff
@@ -116,10 +116,22 @@ This part of the documentation is about the currently developed unified API for
...
+
+    frontend/index
+
+.. include:: /book/frontend/map.rst.inc
+
 Themes
 ------
 
-Here you will learn basics about the Theming concept of Sylius. How to change the theme of your shop? keep reading!
+here you will learn basics about the theming concept of sylius. how to change the theme of your shop? keep reading!
```
Comment (GSadee) [inline docs/book/index.rst]:
```suggestion
Here you will learn basics about the theming concept of sylius. How to change the theme of your shop? keep reading!
```

File: docs/cookbook/frontend/migrating-to-webpack.rst [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+**8.** If you have the following entry in your ``config/packages/_sylius.yaml`` file (available from ``Sylius 1.12``) remove it:
+
+.. code-block:: yaml
+
+    sylius_ui:
+        use_webpack: false
+
+Remove it or change its value to ``true``.
+
+**9.** If you are using Github Actions or any other CI tool, make sure your workflow is using ``yarn build`` or ``yarn build:prod`` command.
```
Comment (GSadee) [inline docs/cookbook/frontend/migrating-to-webpack.rst]:
```suggestion
**9.** If you are using GitHub Actions or any other CI tool, make sure your workflow is using ``yarn build`` or ``yarn build:prod`` command.
```

File: docs/cookbook/frontend/staying-with-gulp.rst [hunk 8]
```diff
@@ -0,0 +1,55 @@
+How to stay with Gulp after Sylius 1.12 update
+=========
```
Comment (GSadee) [inline docs/cookbook/frontend/staying-with-gulp.rst]: Too short line

File: docs/docker-compose.yml [hunk 9]
```diff
@@ -11,7 +11,7 @@ services:
         volumes:
             - ./build:/usr/share/nginx/html:delegated
         ports:
-            - 80:80
+            - 81:80
```
Comment (GSadee) [inline docs/docker-compose.yml]: I'm not sure if it should be changed in this PR

**Review by GSadee — APPROVED** [review]

File: docs/book/frontend/index.rst [hunk 10]
```diff
@@ -0,0 +1,13 @@
+Frontend
+=========
+
+This chapter will introduce you to the world of Sylius' frontend.
+
+
```
Comment (GSadee) [inline docs/book/frontend/index.rst:6]:
```suggestion
```

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Jakub! :tada:

**Pull request merged**